### PR TITLE
Bug 1134711 - Can't edit GPS coordinates of Custom Location+autolander

### PR DESCRIPTION
**Pull request by martapiekarska**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[martasect martasect](https://github.com/martasect) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=fce3df63796c)

**Comment by try-server-hook** [comment]
[martasect martasect](https://github.com/martasect) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f5e676384066)

File: apps/privacy-panel/resources/countries.json [hunk 1]
```diff
@@ -1,4 +1,12 @@
 {
+  "":{
+   "": {
+     "lat" : null ,
+     "lon" : null ,
+     "city": ""
+   }
+  }, 
+
```
Comment (jankeromnes) [inline apps/privacy-panel/resources/countries.json]: Nit: Please remove this empty line, and the trailing space on the line before.

File: apps/privacy-panel/style/lists.css [hunk 2]
```diff
@@ -269,32 +269,32 @@ ul li p + label:not([for]) {
  * Range inputs
  */
 
-input[type=range] {
+input {
```
Comment (jankeromnes) [inline apps/privacy-panel/style/lists.css]: Please don't generalize all these specific `input[type=range]` rules copied from `apps/settings/style/lists.css`, there is no need to do that.

File: apps/privacy-panel/style/lists.css [hunk 3]
```diff
@@ -269,32 +269,32 @@ ul li p + label:not([for]) {
...
   -moz-appearance: none;
   border: none;
   background: none;
 }
 
-input[type=range]:-moz-focusring {
+input:-moz-focusring {
```
Comment (jankeromnes) [inline apps/privacy-panel/style/lists.css]: Same as above.

File: apps/privacy-panel/style/lists.css [hunk 4]
```diff
@@ -269,32 +269,32 @@ ul li p + label:not([for]) {
...
   outline: none;
 }
 
-ul li > label input[type="range"] {
+ul li > label input {
```
Comment (jankeromnes) [inline apps/privacy-panel/style/lists.css]: Same as above.

File: apps/privacy-panel/style/lists.css [hunk 5]
```diff
@@ -269,32 +269,32 @@ ul li p + label:not([for]) {
...
   height: 3rem;
   width: calc(100% - 15rem);
 }
 
-input[type=range]::-moz-range-track {
+input::-moz-range-track {
```
Comment (jankeromnes) [inline apps/privacy-panel/style/lists.css]: Same as above.

File: apps/privacy-panel/style/panels.css [hunk 6]
```diff
@@ -19,12 +19,7 @@
   -moz-user-select: none;
 }
 
-input[type="email"],
-input[type="password"],
-input[type="search"],
-input[type="tel"],
-input[type="text"],
-input[type="url"],
+input,
```
Comment (jankeromnes) [inline apps/privacy-panel/style/panels.css]: This is the correct fix. Thanks!

File: apps/privacy-panel/style/panels.css [hunk 7]
```diff
@@ -44,7 +39,7 @@ ul li:first-child {
   border-top: none;
 }
 
-input[type="range"].blur-slider {
+input.blur-slider {
```
Comment (jankeromnes) [inline apps/privacy-panel/style/panels.css]: No need for this change.

**Comment by try-server-hook** [comment]
[martasect martasect](https://github.com/martasect) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=dc6fec202223)

**Comment by jankeromnes** [comment]
Thanks Marta! Merged as aee8084a4fd30e91717f2b85a32d9ddc2f57f1a4.

**Pull request closed without merging**